pandas/plotting/_core.py

```diff
@@ -2938,26 +2938,59 @@ def scatter(self, x, y, s=None, c=None, **kwds):
     def hexbin(self, x, y, C=None, reduce_C_function=None, gridsize=None,
                **kwds):
         """
-        Hexbin plot
+        Make hexagonal binning plot.
+
+        Make an hexagonal binning plot of `x` versus `y`. If `C` is `None`
+        (the default), this is an histogram of the number of occurrences
+        of the observations at (x[i],y[i]).
```

Comment:
I believe we should quote this using **double** backticks since it's code and not just a variable name:
```
of the observations at ``(x[i],y[i])``.
```
But I cannot find this in the guide. @datapythonista?

Reply:
We discussed the backticks really last minute, and the guide wasn't very clear, but I think that's the standard in general, yes.

As it's code, we can also respect PEP-8 and have the space after the comma. :)